### PR TITLE
WIP: Op progress

### PR DESCRIPTION
**Pull request by duke7553**
TBA

### CODE REVIEW

File: Files/Interacts/Interaction.cs [hunk 1]
```diff
@@ -599,12 +600,17 @@ public async void DeleteItem_Click(object sender, RoutedEventArgs e)
...
+                        0, 
+                        UserControls.StatusBanner.StatusBannerSeverity.Ongoing, 
+                        UserControls.StatusBanner.StatusBannerOperation.Delete);
                 }
 
                 foreach (ListedItem storItem in selectedItems)
                 {
-                    if (selectedItems.Count > 3) { (App.CurrentInstance as ModernShellPage).UpdateProgressFlyout(InteractionOperationType.DeleteItems, ++itemsDeleted, selectedItems.Count); }
+                    uint progressValue = (uint)(itemsDeleted * 100.0 / selectedItems.Count);
+                    if (selectedItems.Count > 3) { banner = banner.UpdateProgress(progressValue); }
```
Comment (hez2010) [inline Files/Interacts/Interaction.cs:613]: I think `IProgress<T>` can be used for normalize progress reporting without implementing a `UpdateProgress()`.
https://docs.microsoft.com/en-us/dotnet/api/system.iprogress-1

**Comment by duke7553** [comment]
If any of the storage item operations become wonky, we can blame this one.

**Comment by XPoppyX** [comment]
is an operation progress window?

**Comment by duke7553** [comment]
@XPoppyX We're not quite ready to announce this feature yet, but it will look something like this when we start rolling it out: 
image

**Pull request merged**